docs/advanced.rst

```diff
@@ -628,10 +629,50 @@ Python side:
 
     py::implicitly_convertible<A, B>();
 
-.. note::
+With this statement, our Python code can now call ``func(a)`` and have it
+treated as if we had written ``func(B(a))``.
+
+When including the additional header file :file:`pybind11/implicit.h`, pybind11
```

Comment:
Whoops, yeah.  Also the example18 reference was wrong, and the class names (A, B) used in the given example were misleading.  Fixed in 6222136.